docs/index.md

```diff
@@ -35,6 +35,14 @@ non-public development, custom hardware handled by vendor
   source hardware and open instruction set architecture. Feel free to check the
   website about recent vPub plans or subscribe to newsletter.
 
+## Newcommers
+
+If you have no prior experience with coreboot, it's highly recommended to visit
+OpenSecurityTraining2 and finish
+[this course](https://p.ost2.fyi/courses/course-v1:OpenSecurityTraining2+Arch4031_x86-64_RV_coreboot+2021_v1/about).
+OpenSecurityTraining2 provides other great courses related to firmware, which
+are freely avialable [here](https://p.ost2.fyi/courses).
```

Comment:
@Stojek139808 this may be good for this page, but I expected more of "Getting started with Dasharo" content, where newcomer get step-by-step procedure for which final goal would be high quality contribution to Dasharo and/or other open-source firmware projects (including documentation, test results, issue reporting).

Above paragraph is too minimalistic to achieve the goal. Please consider yourself as newcomer to open-source firmware (in this case Dasharo), what are the steps you have to take to become contributing community member? And I don't ask here to write all documentation. I ask more for correct set of links with description and given order.

Also Arch4021 also should be mentioned here since we use UefiPayloadPkg from EDKII

Reply:
Now assuming the reader does everything, he should know the basics of coreboot, know how to build it, know how to contribute and also where to ask questions (Matrix).